### DISCUSSION TITLE
Failed to send e-mail for job:xxxxxx: <email address>: Error: unable to verify the first certificate

### DISCUSSION THREAD

**Discussion by GitArnaud44** [jhuckaby/Cronicle#824] [Q&A] [answered]
Mail is not working anymore.
I get the following error when an event sends an email : "Failed to send e-mail for job:xxxxxx: <email address>: Error: unable to verify the first certificate".
I can send a email from the primary and worker from Linux (mailx / sendmail).

It worked before and I don't think I've changed my email configuration.
Do you have any clues?

**Comment by Halfaxas** [comment, marked as answer]
Same issue here :( Did you manage to find a fix?

**Reply by GitArnaud44** [reply]
Hi,
I changed the config to use "external" mail (not in java) in config.json :

        "mail_options": {
                "sendmail": true,
                "newline": "unix",
                "path": "/usr/sbin/sendmail"
        },
